README.md

```diff
@@ -169,20 +169,29 @@ enable('<YOUR INSTRUMENTATION KEY GOES HERE>')
 raise Exception('Boom!')
 ```
 
-**Logging requests**
+**Integrating with Flask**
 ```python
 from flask import Flask
-from applicationinsights.requests import WSGIApplication
+from applicationinsights.flask.ext import AppInsights
 
-# instantiate the Flask application and wrap its WSGI application
+# instantiate the Flask application
 app = Flask(__name__)
-app.wsgi_app = WSGIApplication('<YOUR INSTRUMENTATION KEY GOES HERE>', app.wsgi_app)
+app.config['APPLICATION_INSIGHTS_KEY'] = '<YOUR INSTRUMENTATION KEY GOES HERE>'
+
+# log requests, traces and exceptions to the Application Insights service
+appinsights = AppInsights(app)
```

Comment:
It will also be good to configure EndpointUrl here. Endpoint URL is used for reverse proxy and compliance scenarios.

Reply:
Good to know, done in aa0d8da.